README.md

```diff
@@ -60,6 +62,47 @@ If you implement anything from bonus part, add this information to the README
     - If sorting is not selected or not implementing it should be possible to manage notes order
       using drag and drop
 
+## Part 2: BlockChain task
+
+If you are applying for a position connected with blockchain you should additionally implement this
+task. It adds functionality to your note app which use blockchain wallet.
+
+### Technical requirements
+
+1. Use [Ethereum](https://ethereum.org/en/), [Metamask](https://metamask.io/) wallet
+   and [ethers](https://www.npmjs.com/package/ethers) library for this task.
+    - If you are familiar with other set of technologies you are also free to use them, but in this
+      case write information about it in README
+
+### Acceptance Criteria
+
+Items with __*__ have lower priority than others. Making two blocks without __*__ items is better
+than only one with all items.
+
+1. Login with wallet
+    - It should be possible to login to the note app using you blockchain account.
+    - Notes are independent on different accounts.
+    - \* Logout, account and network should work without reload.
```

Comment:
@sashasashasasha151 I think it is better to move this `*` into a separate section, example:
```
3. Other (Lower priority)
- Logout, account and network...
- After page reload all notes with not finished transactions...
```

This way, it is less confusing, and it would be easier to give scores as well. Also if it is lower priority, should we move this to the `Bonus` part instead?